Assets/Scripts/BeamMovement.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BeamMovement : MonoBehaviour
+{
+    public static float speed = 10.0f;
+    public static float maxSpeed = 70.0f;
+
+    // Update is called once per frame
+    void Update()
+    {
+        //new Vector3(0,1,0) same as Vector3.up
+        //need time.deltatime with speed. timedeltatime(time between frames) averages frames so fast computers dont go faster
+        //space.world is to move it relative to world instead of the object
+
+        //yo david its evan gabriel said you would give her shit for having notes in comments. i told her its ok. :)
+        //if youre not david then ignore the previous comment ;)
```

Comment:
😆

Reply:
lol I am confused :)

Reply:
You can have notes in comments. I would say that step 1 is to have good names, step 2 is to add summaries. Then, if necessary, feel free to explain further in comments. But I think steps 1 and 2 will clarify most things if done well.

Reply:
haha I didn't know if you would be cool with me writing actual notes about programming in my project, and I was talking with Evan Karlson and he told me i could and to put that in there :)